test_nested_virtualization.py

```diff
@@ -0,0 +1,93 @@
+"""
+
+Nested Virtualization Acceptance Tests
+====================
+
+Customers can start virtual servers within Plus and Flex servers.
```

Comment:
```suggestion
Customers can start virtual servers inside VMs (nested virtualiziation).
```

-> This is not just a Plus and Flex thing (or not for much longer anyway).